script.js

```diff
@@ -0,0 +1,224 @@
+        <span class="subheading-list">${workArray.subheadlist1}</span>
+        <img src="./Icons/${workArray.counter}" alt="" />
+        <span class="subheading-list">${workArray.subheadlist2}</span>
+      </div>
+      <img class="detail-img" src="./Icons/desktop/${workArray.imageName}" alt="">
+      <div class="details-content">
+        <p class="description">
+        ${workArray.des2}
+        </p>
+        <div class="details-right">
+          <ul class="web-languages">
+          ${workArray.languages.map((lang) => `<li><a href="/">${lang}</a></li>`).join('')
+}
+          </ul>
+          <div class="details-links">
+            <a href="${workArray.liveLink}">
+              <button>
+                <p>See live</p> <img src="./Icons/social/Icon - Export.svg" alt="">
+              </button>
+            </a>
+            <a href="${workArray.sourceLink}">
+              <button>
+                <p>See source</p> <img src="./Icons/social/Frame.svg" alt="">
+              </button>
+            </a>
+          </div>
+        </div>
+      </div>
+    </div>`;
+
+      modal.classList.remove('hidden');
+      modal.innerHTML = modalHtml;
+
+      const closePopup = document.querySelector('.closePopup');
+      closePopup.addEventListener('click', () => {
+        modal.classList.add('hidden');
+      });
+    });
+  });
+}
+
+const showbtn = document.querySelector('.active');
+const hidebtn = document.querySelector('.navbar');
+const popup = document.querySelector('.project-detail');
+
+const x = document.getElementById('menubar');
+const show = () => {
+  x.classList = 'active';
+};
+
+const hide = () => {
+  x.classList = 'navbar';
+};
+
+const remove = () => {
+  popup.classList.remove('hidden');
+};
+
+showbtn.addEventListener('click', show);
+hidebtn.addEventListener('click', hide);
+popup.addEventListener('click', remove);
+
+// contact form
+
+function showPopup() {
+  const form = document.getElementById('contact-form');
+  const emailInput = form.querySelector('input[name="email"]');
+  const errorMessage = document.getElementById('error-message');
+
+  document.addEventListener('DOMContentLoaded', () => {
+    const savedFormData = localStorage.getItem('formData');
+    if (savedFormData) {
+      const parsedFormData = JSON.parse(savedFormData);
+      form.firstname.value = parsedFormData.firstname;
+      emailInput.email.value = parsedFormData.emailInput;
+      form.msg.value = parsedFormData.msg;
+    }
+  });
+
+  form.addEventListener('change', () => {
+    const formData = {
+      firstname: form.firstname.value,
+      email: form.email.value,
+      msg: form.msg.value,
+    };
+    localStorage.setItem('formData', JSON.stringify(formData));
+  });
+
+  form.addEventListener('submit', (event) => {
+    event.preventDefault();
+
+    const formData = {
+      firstname: form.firstname.value,
+      email: form.email.value,
+      msg: form.msg.value,
+    };
```

Comment:
- [ ] As mentioned by the previous reviewer :point_up: the contact form information is persistent to page reloads. After filling out the form, and not submitting it, the entered values should be saved in the browser. This is not happening when filling out the form for your project. Kindly fix the issue.

_**NOTE: Please watch this video on Chrome browser I don't know why this video is not played on other browsers so kindly use Chrome browser**_

[screen-capture (22).webm](https://github.com/abdullah97-pro/Portfolio-Mobile-Project/assets/45226848/a71d2827-7b26-4b9a-84e7-97134d57d71b)